modules/101-cert-manager/docs/FAQ_RU.md

```diff
@@ -58,6 +58,71 @@ default            example-com                     13m
+
+## Как добавить дополнительный `ClusterIssuer`?
+
+### В каких случаях требуется дополнительный `ClusterIssuer`
+
+В стандартной поставке присутствуют два `ClusterIssuer`, которые издают сертификаты из доверенного публичного удостоверяющего центра LetsEncrypt - `letsencrypt` и `letsencrypt-staging`.
+Чтобы издать сертификаты на доменное имя через LetsEncrypt, сервис требует осуществить подтверждение владения доменом. 
+Cert-manager поддерживает несколько методов для подтверждения владения доменом при использовании `ACME`(Automated Certificate Management Environment):
+* `HTTP-01` -  метод подразумевает, что cert-manager создаст временный Pod в кластере, который будет слушать на определенном URL для подтверждения владения доменом. Для его работы необходимо иметь возможность направлять внешний трафик на этот Pod, обычно через `Ingress`.
+* `DNS-01` -  метод подразумевает, что cert-manager делает запись TXT запись в DNS для подтверждения владения доменом. У cert-manager есть встроенная поддержка популярных провайдеров DNS: AWS Route53, Google Cloud DNS, Cloudflare, и другие.
+
+`ClusterIssuers` присутствующие в поставке используют метод `HTTP-01`.
+
+Таким образом дополнительный `ClusterIssuer` может потребоваться в следующих случаях:
+1. Требуется издание сертификатов в удостоверяющем центре, отличным от LetsEncrypt (в т.ч. в приватном). Поддерживаемые УЦ доступны [в документации cert-manager](https://cert-manager.io/docs/configuration/)
+2. Требуется создание сертификата через LetsEncrypt, но через метод `DNS-01`
+   1. Требуется издание wildcard сертификатов (доступно только через метод `DNS-01`)
+
+### Как добавить дополнительный `ClusterIssuer` использующий LetsEncrypt и метод подтверждения `DNS-01`
+
+Для подтверждения владения доменом через LetsEncrypt с помощью метода `DNS-01` требуется настроить возможность создания TXT-записей в публичном DNS. 
+
+У `сert-manager` есть поддержка механизмов для создания TXT-записей в некоторых популярных DNS: AzureDNS, Cloudflare, Google Cloud DNS, и другие. Полный перечень доступен [в документации cert-manager](https://cert-manager.io/docs/configuration/acme/dns01/)
+
+Пример использования AWS Route53 доступен в разделе [`Как защитить учетные данные cert-manager`](#как-защитить-учетные-данные-cert-manager)
+
+Использование сторонних DNS-провайдеров реализуется через метод `webhook`.  Когда cert-manager выполняет вызов `ACME` `DNS01`, он отправляет вызов на webhook-сервер, который затем выполняет нужные операции для обновления записи DNS.
+При использовании данного метода требуется разместить сервис, который будет обрабатывать хук и осуществлять создание TXT записи в DNS-провайдере.
+
+В качестве примера рассмотрим использование сервиса Yandex Cloud DNS
+
+Для обработки webhook предварительно разместим в кластере сервис `Yandex Cloud DNS ACME webhook` согласно [официальной документации](https://github.com/yandex-cloud/cert-manager-webhook-yandex)
```

Comment:
I think the part about YC is good enough and we do not need to dive into all the nuances of installation of the webhook.